### PR TITLE
HW5: Task 1

### PR DESCRIPTION
**Pull request by ValeriL**

### CODE REVIEW

File: Homework/homework5/task1.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+
+class Homework:
+    def __init__(self, text: str, deadline: int):
+        self.text = text
+        self.deadline = datetime.timedelta(days=deadline)
+        self.created = datetime.datetime.now()
+
+    def is_active(self) -> bool:
+        return (self.created + self.deadline) > datetime.datetime.now()
```
Comment (Cjkjvfnby) [inline Homework/homework5/task1.py:46]: What is the precedence for `+` and `>`?

File: Homework/homework5/task1.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+class Teacher:
+    def __init__(self, first_name: str, last_name: str):
+        self.first_name = first_name
+        self.last_name = last_name
+
+    def create_homework(self, text: str, days: int) -> Homework:
+        return Homework(text, days)
+
+
+if __name__ == "__main__":
```
Comment (Cjkjvfnby) [inline Homework/homework5/task1.py:70]: You don't need this, since you have tests.

File: Homework/homework5/test_task1.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+from homework5.task1 import Homework, Student, Teacher
+
+
+def test_homework_is_active_for_not_expired_homework():
+    hw_text, days = "Any homework", 3
```
Comment (Cjkjvfnby) [inline Homework/homework5/test_task1.py:5]: You can inline these variables.

File: Homework/homework5/test_task1.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+
+def test_do_homework_for_not_expired_homework():
+    hw_text, days = "Any homework", 3
+    not_expired_homework = Homework(hw_text, days)
+
+    first_name, last_name = "Roman", "Petrov"
+    student = Student(first_name, last_name)
+
+    assert student.do_homework(not_expired_homework) == not_expired_homework
```
Comment (Cjkjvfnby) [inline Homework/homework5/test_task1.py:32]: What else can you test here?

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: Homework/homework5/task1.py (hunk 1 above)
Comment (ValeriL) [reply Homework/homework5/task1.py:46]: + precedence is higher so without parentheses

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**